DevCycle.SDK.Server.Local/Api/LocalBucketing.cs

```diff
@@ -20,23 +20,31 @@ public LocalBucketingException(string message): base(message)
         }
         
     }
-    
-    public class LocalBucketing : ILocalBucketing
+
+    public class LocalBucketing
     {
         private static readonly SemaphoreSlim WasmMutex = new(1, 1);
         private static readonly SemaphoreSlim FlushMutex = new(1, 1);
         private Func<string, string> handleError;
+        
+        private Dictionary<string, int> sdkKeyAddresses;
+
+        private List<int> pinnedAddresses;
```

Comment:
This should probably be a HashSet instead of a list - https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.hashset-1?view=net-7.0